neuralmagic/benchmarks/configs/benchmark_serving.json

```diff
@@ -0,0 +1,20 @@
+{
```

Comment:
These configs define what bechmark command(s) to run. 
we do a `itertools.product(script_args.values())` to get all the combinations and each combination is run as a separate benchmark.